content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -173,7 +180,7 @@ export class _CollapsibleSection extends React.PureComponent {
                 {learnMore &&
                   <span className="learn-more-link">
                     <a href={learnMore.link.href}>
-                      <FormattedMessage id={learnMore.link.id} />
+                      <span data-l10n-id={learnMore.link.id} />
```

Comment:
Ah... Can switch to data-l10n-name here

Reply:
This might work 

```
<span className="learn-more-link" data-l10n-id={learnMore.link.id}><a data-l10n-name=“learn-more” href={learnMore.link.href}>How it works</a>
</span>
```
where newtab-pocket-how-it-works in ftl is
`newtab-pocket-how-it-works = <a data-l10n-name=“learn-more”> How it works </a>`

Will be good to check with other locales or[ intl.l10n.pseudo set to accented](https://firefox-source-docs.mozilla.org/intl/l10n/l10n/fluent_tutorial.html#pseudolocalization)